app/views/exam_templates/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<% content_for :title, ExamTemplate.model_name.human.pluralize %>
+
+<%= render partial: 'form',
+           locals: { exam_template: @exam_templates }%>
```

Comment:
local variable exam_template becomes nil when I try @exam_templates[0]. Should I leave it?